src/test/kotlin/org/rust/ide/intentions/NestUseStatementsIntentionTest.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+class NestUseStatementsIntentionTest : RsIntentionTestBase(NestUseStatementsIntention()) {
```

Comment:
Too few tests. At least:
1. try o places caret to different places
2. more use items with different base paths (prefixes)
3. some of use items are already nested
4. use items with comments (we should preserve comments)
5. Check that intention isn't available if nothing to nest